.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+    - 'master'
+    tags:
+    - 'v*'
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
+  ubuntu18:
+    name: ubuntu 18.04
```

Comment:
could we make this "latest ubuntu/fedora/whatever" rather than 18.04?

Reply:
I am usually not in favor of "latest", because that may break when it changes. I could fix it to 20.04, it's just that I wanted to reproduce the current travis build, which I believe is on 18.04.

Your call: 20.04 or should I try "latest"?

Reply:
if there is a "latest" that always point to the latest version, better. That way we don't need to keep updating it.

I like to live on bleeding edge. If it breaks, we should fix because sooner or later someone will be using that version.